network/node1/config.toml

```diff
@@ -0,0 +1,19 @@
+# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+
+proxy_app = "tcp://bdb-one:46658"
```

Comment:
again hard coded configs should be part of the container build scripts we can configure host and seed in entrypoint script

Reply:
I do not understand the concern here. It seems to me fairly reasonable to provision a configuration file. This is really what this is about.